bundles/io.openliberty.tools.eclipse.lsp4e/plugin.xml

```diff
@@ -89,5 +125,22 @@
             id="io.openliberty.tools.eclipse.org.microprofile.mpserver">
       </contentTypeMapping>  
    </extension>
+   
+      <extension
+         point="org.eclipse.lsp4e.languageServer">
+      <server
+            class="io.openliberty.tools.eclipse.liberty.languageserver.LibertyLSConnection"
+            id="io.openliberty.tools.eclipse.org.liberty.languageserver"
```

Comment:
Can you open an issue for the naming of this?  "Liberty Config LS" doesn't sound wrong just thinking we should socialize.